server/services/participants.js

```diff
@@ -121,9 +121,19 @@ const getWithdrawnParticipantsBySite = async (siteID) => {
 };
 
 const getParticipantByID = async (id) => {
```

Comment:
I'd like either a function name change to indicate that it's returning statuses as well, and/or function comment with what it does